release-notes/swan-lake-alpha3-release-note.md

```diff
@@ -31,20 +18,26 @@ If you are already using Ballerina, you can directly update your distribution to
 If you are using an **Update Tool version below 0.8.14**, execute the `ballerina update` command to update it. Next, execute the command below to update to Swan Lake Alpha3.
 
 > `bal dist pull slalpha3`
+<br><br>
 
-#### For New Users
+### Installing Ballerina
 
-If you have not installed Ballerina, then download the [installers](/downloads/#swanlake) to install.
-
-### Highlights
+If you haven't installed Ballerina, download the [installers](/downloads/#swanlake) to install. <br><br>
```

Comment:
```suggestion
If you haven't installed Ballerina, download the [installers](/downloads/#swanlake) to install. <br><br>
```
We do not use shortened forms of words with the apostrophe in technical documentation.